usaspending_api/search/v2/urls_visualizations.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
```

Comment:
Is this file necessary? the endpoint it `api/v2/search/spending_by_category`, so we may not need this.